QandA/views.py

```diff
@@ -0,0 +1,6 @@
+
+# Create your views here.
+from django.http import HttpResponse
+
+def index(request):
+    return HttpResponse("Hello,World! This is Prabal")
```

Comment:
- Line break at end.
- Also, why do we need this view?

Reply:
Removed this view, was just trying to learn how to write 'views' in django.